libr/core/disasm.c

```diff
@@ -731,9 +731,17 @@ static void ds_build_op_str(RDisasmState *ds) {
 					core->parser->flagspace = -1;
 				}
 			}
+			RCore *core = ds->core;
+			ut64 p = ds->analop.ptr;
+			char *msg = calloc (sizeof (char), 128); // is this big enough?
+			r_io_read_at (core->io, p, (ut8 *)msg, 128 - 1);
+			ut64 num = r_read_ble (msg, core->print->big_endian, ds->analop.refptr * 8);
```

Comment:
We should check if ds->analop.refptr have a valid value before multiplying it (plugins can put a negative value or a huge one producing undesired effects